### PR TITLE
[Desire Lines] Added new advancements

### PR DESCRIPTION
**Pull request by AndanteDevs**
- Added 3 new advancements
    - `I Would Walk 500 Chunks...`: Ruin 8000 blocks to coarse dirt; you'll find those oak planks
    - `That's a Lot of Damage`: Ruin 10000 blocks to coarse dirt; how about a little more?
- Fixed an incorrect notation

### CODE REVIEW

**Comment by ToffeeMax** [comment]
I like it - but not sure on some of the names?

**Comment by AndanteDevs** [comment]
Thanks for that @ToffeeMax x3

**Comment by AndanteDevs** [comment]
Just to note, the scoreboard `gm4_affcoarse` means affected coarse dirt.

File: gm4_desire_lines/data/desire_lines/functions/path.mcfunction [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+#check blocks
 fill ~ ~-1 ~ ~ ~-1 ~ coarse_dirt replace dirt
 fill ~ ~-1 ~ ~ ~-1 ~ dirt replace grass_block
 fill ~ ~ ~ ~ ~ ~ air replace snow[layers=1]
 fill ~ ~ ~ ~ ~ ~ snow[layers=1] replace snow[layers=2]
 execute if block ~ ~ ~ #desire_lines:breakable run setblock ~ ~ ~ air destroy
+#advancement check
+execute if block ~ ~-1 ~ coarse_dirt run scoreboard players add @s gm4_dl_affcoarse 1
+execute if score @s gm4_dl_affcoarse matches 1000.. run advancement grant @s only gm4:desire_lines_1000
+execute if score @s gm4_dl_affcoarse matches 10000.. run advancement grant @s only gm4:desire_lines_10000
```
Comment (misode) [inline gm4_desire_lines/data/desire_lines/functions/path.mcfunction]: These ranges can be removed as it can only ever be increased by 1

**Comment by misode** [comment]
5 advancements for this simple module seems a bit much imo

**Comment by AndanteDevs** [comment]
Agreed

**Comment by sparks0** [comment]
I like this but my one worry is how long it would take to ruin 1,000,000 blocks for achievement hunters. I guess it's something that stacks up over time while you're doing other things but I have a feeling the time spent walking to reach a million is gonna be measured in hours or days... Haven't done the maths on that though.

**Comment by Bluefire-610** [comment]
I'm doing the maths.

**Comment by Kroppeb** [comment]
A 1000000 seconds is 11.5 days so if you were to destroy 1 block every 5 seconds (idk what actual speed is) and you are online 6 hour a day it would take >230 days

**Comment by Bluefire-610** [comment]
This is a rough estimate. Let's say desire lines turns 25% of the blocks sprinted over into dirt (it's way less than that). That means it would take 197hrs of running to complete the advancement.

**Pull request merged**